src/popup/popup.js

```diff
@@ -96,11 +103,11 @@ function handleAfterLoginAttempt(options) {
         document.getElementById("login-status-success").classList.remove('hidden');
         document.getElementById("login-status-fail").classList.add('hidden');
         document.getElementById("login-button").classList.add("d-none");
-        document.getElementById('logout-button').classList.remove("d-none");
+        document.getElementById('logout-button-div').classList.remove("d-none");
     } else {
         document.getElementById('login-fail-error').innerText = error + '.';
         document.getElementById("login-status-fail").classList.remove('hidden');
-        document.getElementById("login-status-success").classList.add('hidden');
+        document.getElementById("login-status-success").classList.add('hiddenF');
```

Comment:
`hiddenF` instead of `hidden`. Can this even work? 😄

Reply:
This was a typo. Fixed in 94c86e9db649a52cd9d0c518b064deede7942af3